src/components/Header.jsx

```diff
@@ -10,7 +11,7 @@ class Header extends Component {
     return (
       <div className="header-container">
         <div>
-          <img className="logo" src="/logo.png" alt="" />
+          <img className="logo" src={`/${logo}`} alt="Logo" />
```

Comment:
LOL you had to do this to get it to work? lmao that is very unfortunate.

Reply:
It be like sometime.